src/app/services/baw-api/security/security.service.ts

```diff
@@ -209,7 +221,7 @@ export class SecurityService {
     this.sessionDetails()
       .pipe(
         tap((user) => (authToken = user.authToken)),
-        mergeMap(() => this.userService.show()),
+        mergeMap(() => this.userService.showWithoutNotification()),
```

Comment:
seems unbalanced, you just pass the option above in `destroy` why a special helper method here?

Reply:
I cannot update the inputs of the `UserService`'s `show` method without causing significant churn. So instead i just created a `showWithoutNotification` method to be used by the `SecurityService`. In the case of the `signOut` function, there is no scenario where I would want to show the failure notification, because even if failure occurs, I will just clear the cookie and the auth token which achieves similar results.

Reply:
One could argue that i shouldn't attempt to forcefully signout the user on the clientside only if the api fails to respond, and as such should create a `signOutWithoutNofication` method